game/src/main/scala/io/aigar/game/EntityContainer.scala

```diff
@@ -0,0 +1,22 @@
+package io.aigar.game
+
+import com.github.jpbetz.subspace.Vector2
+
+
+trait EntityContainer {
+  def respawn(grid: Grid, players: List[Player], respawnRetryAttempts: Int): Option[Vector2] = {
```

Comment:
Also, could this always return something (i.e. `Vector2` instead of `Option[Vector2]`? I thought the goal was "try to respawn N times, but if you fail, give me any position"
I was envisioning something along the lines of trying to change the position N times or else return the last generated one.